app/views/browse/deployment-config.html

```diff
@@ -89,6 +97,21 @@
                   <uib-tab active="selectedTab.details">
                     <uib-tab-heading>Details</uib-tab-heading>
                     <div class="resource-details" ng-if="deploymentConfig">
+                      <div ng-if="deploymentConfig.spec.paused || updatingPausedState" class="alert alert-info animate-if">
+                        <span class="pficon pficon-info" aria-hidden="true"></span>
+                        <span ng-if="updatingPausedState && !deploymentConfig.spec.paused" class="animate-if">
+                          Pausing deployments...
```

Comment:
It took long enough that I wondered if it was working for deployment configs. Deployments seem faster.